src/site/content/en/blog/fast-playback-with-preload/index.md

```diff
@@ -0,0 +1,599 @@
+---
```

Comment:
I'm OK with us using the URL `web.dev/video-preload` for this article

Reply:
Although since you mention that it also works with `<audio>`, perhaps that name is too narrow?

Reply:
I made the intro and title a little more generic.